app/controllers/spaces.rb

```diff
@@ -0,0 +1,64 @@
+class MakersBnb < Sinatra::Base
+
+  get '/edit/:space_id' do
+    unless current_user
+      redirect '/sessions/new'
+    end
```

Comment:
Since this appears quite often, could it be extracted elsewhere? Such as reopening `Sinatra::Base`, defining a class method `verify_user_exists` that does this, and then calling `verify_user_exists` all over the shop instead of this?